src/coreclr/src/vm/i386/stublinkerx86.cpp

```diff
@@ -3845,7 +3845,7 @@ VOID StubLinkerCPU::EmitShuffleThunk(ShuffleEntry *pShuffleEntryArray)
 
     UINT16 emptySpot = 0x4 | ShuffleEntry::REGMASK;
 
-    while (true)
+    for (;;)
```

Comment:
Why was this change necessary? `while (true)` is preferred over `for (;;)` because it is clearer and avoids unnecessary indentation changes (`for (; ;)` vs. `for (; ; )`).